client/blocks/eligibility-warnings/hold-list.tsx

```diff
@@ -155,7 +156,12 @@ export const HardBlockingNotice = ( {
 	const blockingHold = holds.find( ( h ): h is keyof ReturnType< typeof getBlockingMessages > =>
 		isHardBlockingHoldType( h, blockingMessages )
 	);
-	if ( ! blockingHold ) {
+
+	/*
+		This accounts for Atomic sites that are on a plan below Business so we can prompt them to upgrade
+		instead of showing them an "Upload in progress" notice.
+	*/
+	if ( ! blockingHold || isAtomicSiteWithoutBusinessPlan( holds ) ) {
```

Comment:
Can you please expand on why / where is that needed? Shouldn't we also check for eCommerce Plans Could we reuse `siteCanUploadThemesOrPlugins` selector?  

Also, this block renders in several places. Should it affect them?

Reply:
>could we reuse `siteCanUploadThemesOrPlugins` selector

AFAIK no. The `siteCanUploadThemesOrPlugins` selector checks it has a business (or above) plan AND it's Atomic. We need to check that it's Atomic and a plan _below_ business.

Reply:
>Also, this block renders in several places. Should it affect them?

Based on the additional logic I've added to this, it's fine... I think.

Reply:
> Can you please expand on why / where is that needed?

Expanded on comments to explain usage in https://github.com/Automattic/wp-calypso/pull/52776/commits/3c581ce89ae2175f6b1a5b9054de293c779b5eb2